client/styled-components/Buttons.jsx

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+const Button = styled.button`
+`;
+
+const FindButton = styled(Button)`
```

Comment:
Does setting 'styled.button' to a const then using styled(Button) add some functionality?
Rather than just using 'styled.button'? Haven't seen this pattern yet.

Reply:
That's a good question, I think I may have been over thinking it. Going to refactor that.